x/concentrated-liquidity/position.go

```diff
@@ -150,3 +151,15 @@ func (k Keeper) CreateFullRangePosition(ctx sdk.Context, concentratedPool types.
 
 	return amount0, amount1, liquidity, joinTime, nil
 }
+
+func CalculateUnderlyingAssetsFromPosition(ctx sdk.Context, position model.Position, pool types.ConcentratedPoolExtension) (sdk.Dec, sdk.Dec, error) {
```

Comment:
nit: we can define this on the pool struct as a method instead of passing in the pool as param